### ISSUE TITLE
Button doesn't have a proper focus indicator on high contrast mode

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#4264]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No
Requested priority: Normal

#### Describe the issue:

On high contrast mode on Edge the button element doesn't have a proper focus indicator. Repros on all buttons. https://developer.microsoft.com/en-us/fabric#/components/panel

#### Actual behavior:
![image with focus on a button](https://user-images.githubusercontent.com/24945574/37373618-8b947f30-26d4-11e8-9047-582ff045b0ae.png)

#### Expected behavior:
Focus indicator should be clearer. Now on focus the border just shrinks a little.

**Comment by lynamemi** [comment]
Thanks for reporting! This one I have a PR out for 😄 
https://github.com/OfficeDev/office-ui-fabric-react/pull/4221
